apiary_v2.apib

```diff
@@ -1258,7 +1431,36 @@ the listed orders by their creation date.
 
     + Body
 
-          {}
+          {
+            "count": 12,
+            "next": "https://factory.unmade.com/v2/orders/?page=2",
+            "previous": null,
+            "results": [
+              {
+                "id": "a38c331d-cc6d-4e03-b514-91e27ae8b02e",
+                "created": "2017-07-31T16:52:50.905026Z",
+                "reference": "mc_testorder6",
+                "num_jobs": 4,
+                "jobs": "https://factory.unmade.com/v2/orders/a882ac66-9374-4910-8c93-9ea45a470bd4/jobs/",
+                "shipping_address": "https://factory.unmade.com/v2/orders/a882ac66-9374-4910-8c93-9ea45a470bd4/shipping_address/",
+                "ecommerce_order": "https://partner.embed.unmade.com/v2/orders/a882ac66-9374-4910-8c93-9ea45a470bd4",
+                "partner_data": {
+                  "my_partner_data_key": "foo"
+                },
+                "attachments": {
+                  "packing_slip": "https://cdn.unmade.com/order/900243544001/attachments/packing_slip.pdf",
+                  "commercial_invoice": "https://cdn.unmade.com/order/900243544001/attachments/import_invoice.pdf"
+                },
```

Comment:
what we decided with the attachments?

Reply:
What do you mean? I think just providing URLs like this is fine?

The attachments with addresses will be hidden if the order isn't ready to ship. That was the only change we've discussed.

Reply:
ah no that's what I mean the ones that need to be hidden will be hidden. And we can figure out if it make sense to have another endpoint to retrieve attachments on its own, or if we hide them if we show the key here and not url or something else